### ISSUE TITLE
Unrecognized key(s) in object: '$schema'

### ISSUE THREAD

**Issue by saiichihashimoto** [chromaui/chromatic-cli#1022]
```
Chromatic CLI v11.6.0
https://www.chromatic.com/docs/cli


✖ Configuration file chromatic.config.json was invalid, please check the allowed keys.

- Unrecognized key(s) in object: '$schema'



→ View the full stacktrace below

If you need help, please chat with us at https://www.chromatic.com/docs/cli for the fastest response.
You can also email the team at support@chromatic.com if chat is not an option.

Please provide us with the above CLI output and the following info:
{
  "timestamp": "2024-08-12T22:31:41.116Z",
  "sessionId": "aae7d813-d95a-414a-8356-8aec5a669236",
  "nodePlatform": "darwin",
  "nodeVersion": "20.16.0",
  "packageName": "chromatic",
  "packageVersion": "11.6.0",
  "flags": {
    "outputDir": [],
    "storybookBuildDir": [],
    "externals": [],
    "onlyStoryFiles": [],
    "onlyStoryNames": [],
    "untraced": [],
    "fileHashing": true,
    "interactive": true,
    "appCode": []
  },
  "errorType": "Error",
  "errorMessage": "✖ Configuration file chromatic.config.json was invalid, please check the allowed keys."
}

Error: ✖ Configuration file chromatic.config.json was invalid, please check the allowed keys.

- Unrecognized key(s) in object: '$schema'
- ```

**Comment by cgbl-90** [comment]
hello @saiichihashimoto, not all options/flags are supported by the schema. Find the list of supported flags here: https://www.chromatic.com/docs/configure/#options Filter by `Config File`.

**Comment by saiichihashimoto** [comment]
@cgbl-90

https://www.chromatic.com/docs/cli/#chromatic-config-file

The chromatic docs are where I got the $schema key from. So either those docs should be changed or the key should be allowed.

**Comment by winkerVSbecks** [comment]
@saiichihashimoto this is an oversight on my end. We haven't updated the CLI to allow for the new `$schema` key. You can remove it to unblock yourself. And I plan to fix that asap.

**Comment by ghengeveld** [comment]
:rocket: Issue was released in [`v11.7.1`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.7.1) :rocket:

**Comment by ghengeveld** [comment]
:rocket: Issue was released in [`v11.7.1`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.7.1) :rocket:

**Comment by ghengeveld** [comment]
:rocket: Issue was released in [`v11.7.1`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.7.1) :rocket:
